docs/index.rst

```diff
@@ -0,0 +1,12 @@
+Python client for the Yola API
+==============================
+
+.. toctree::
+   :maxdepth: 1
+
```

Comment:
There really should be some introduction before we dive into API details. But that can be in a future PR.